modules/network/multivpc/README.md

```diff
@@ -0,0 +1,123 @@
+## Description
+
+This module accomplishes the following:
+
+* Creates N [VPC networks][vpc], where N is inclusively between 2 and 8
+  * There is one subnetwork per VPC
+* Sets up the VPCs to use non-overlapping IP ranges
+* Outputs the `additional_networks` parameter, which is compatible with Slurm
+  modules
+
+There are 4 variables that differentiate this module from the standard VPC
+module.
+
+1. `network_prefix`: This is the name prefix of the VPCs to be created.  All
+   networks and subnetworks will start with this and end with a unique number.
+1. `network_count`: This is the number of VPCs to be created.
+1. `global_ip_address_range`: This is an IP address with a CIDR prefix, which
+   all of the VPCs will be under.
+1. `network_cidr_prefix`: This is the CIDR prefix that defines the address
```

Comment:
Looking at the definition, it is suddenly obvious this is a suffix. It corresponds to a number of bits. I would rename to `subnetwork_cidr_prefix` and 

> The [CIDR suffix](https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing#CIDR_notation) that defines the the number of IP addresses in each subnetwork (smaller suffix results in more IP addresses)

Reply:
Done